fitbenchmarking/fitting/mantid_utils.py

```diff
@@ -130,6 +130,9 @@ def ignore_invalid(prob, cost_function):
     """
 
     ignore_invalid = cost_function == 'Least squares'
+
+    # The WISH data presents some issues
+    # For which this adhoc if must is present
```

Comment:
is it possible to have a little more explanation as to why this error occurs and why the if statement works?

Reply:
Unfortunately, I also have no idea what the specific cause is. I tired looking into it and got nowhere pretty quickly as there seems to be some special thing about the WISH nexus files.

Reply:
ok, can you change the is to a to a be?